src/cmd/append.go

```diff
@@ -19,6 +19,18 @@ type appendConfig struct {
 var appendCommand = &cobra.Command{
 	Use:   "append <branch>",
 	Short: "Creates a new feature branch as a child of the current branch",
+	Long: `Creates a new feature branch as a child of the current branch.
+
+Syncs the current branch,
+forks a new feature branch with the given name off the current branch,
+makes the new branch a child of the current branch,
+pushes the new feature branch to the remote repository,
+and brings over all uncommitted changes to the new feature branch.
+
+Additionally, when there is a remote upstream,
+the main branch is synced with its upstream counterpart.
```

Comment:
I think this should only be documented on the sync command and doesn't need to be repeated here. Also the append command doesn't necessarily sync the main branch (for example if appending to a perennial branch)